processing/src/main/java/io/druid/query/ordering/StringComparator.java

```diff
@@ -17,29 +17,31 @@
  * under the License.
  */
 
-package io.druid.query.search;
```

Comment:
I wonder what made git think that StrlenSearchSortSpecTest.java was renamed to StringComparator.java